Feature/heating si7021 (#3150)
OttoWinter/esphomedocs: components/sensor/htu21d.rst

- **humidity** (*Optional*): The information for the humidity sensor.
  All options from :ref:`Sensor <config-sensor>`.

- **heater** (*Optional*): The information for the heater sensor.
  All options from :ref:`Sensor <config-sensor>`.

- **update_interval** (*Optional*, :ref:`config-time`): The interval to check the sensor. Defaults to ``60s``.

> Heater can be enable to helps to correct the reading, please read the datasheet for more information.

Reviewer: ```suggestion
The heater may be enabled to help correct the reading; see the datasheet for more information.
```
